Sources/MapboxCoreNavigation/EHorizon/RoadShield.swift

```diff
@@ -0,0 +1,42 @@
+import MapboxNavigationNative
+
+/// Describes a road shield information.
+///
+/// - note: The Mapbox Electronic Horizon feature of the Mapbox Navigation SDK is in public beta
+/// and is subject to changes, including its pricing. Use of the feature is subject to the beta product restrictions
+/// in the Mapbox Terms of Service. Mapbox reserves the right to eliminate any free tier or free evaluation offers at
+/// any time and require customers to place an order to purchase the Mapbox Electronic Horizon feature, regardless of
+/// the level of use of the feature.
+public struct RoadShield: Equatable {
+    /// The base url for a shield image.
+    public let baseUrl: String
+
+    /// The shield display reference.
+    public let displayRef: String
+
+    /// The shield text.
+    public let name: String
+
+    /// The shield text color.
```

Comment:
We should probably expand how color is described into a string. Is it it's name, hex string or else? Probably event convert to UIColor?

Reply:
I updated the comment in similar way to [Directions](https://github.com/mapbox/mapbox-directions-swift/blob/1f878617c7df86a51880bc961e85b0055cab2dcd/Sources/MapboxDirections/VisualInstructionComponent.swift#L206)